core/SConscript.firmware

```diff
@@ -506,6 +506,8 @@ PROTO_SOURCES_DIR = '../../../common/protob/'
 exclude_list = [PROTO_SOURCES_DIR + 'messages-bootloader.proto']
 if not THP:
     exclude_list.append(PROTO_SOURCES_DIR + 'messages-thp.proto')
+if PYOPT == '1':
```

Comment:
Nit: consider using `PYOPT != '0'` everywhere so as not to fail when somebody uses `PYOPT=3`.

Reply:
Thanks, fixed in a2df0f3dbb